packages/mongodb-rag-core/src/MongoDbEmbeddedContentStore.test.ts

```diff
@@ -118,7 +121,7 @@ describe("nearest neighbor search", () => {
     minScore: 0.9,
   };
 
-  let store: (DatabaseConnection & EmbeddedContentStore) | undefined;
+  let store: MongoDbEmbeddedContentStore | undefined;
   beforeEach(async () => {
     // Need to use real Atlas connection in order to run vector searches
     const databaseName = MONGODB_DATABASE_NAME;
```

Comment:
make this a randomly generated name please

Reply:
unfortunately we can't here - its testing vector search so it needs to point to the db that has the embedded data. Best course is to just not call `db.drop()` (lol) or maybe add a readonly version of the store and use that in these tests